Modules/_testinternalcapi.c

```diff
@@ -1650,6 +1650,20 @@ get_rare_event_counters(PyObject *self, PyObject *type)
     );
 }
 
+static PyObject *
+reset_rare_event_counters(PyObject *self, PyObject *type)
```

Comment:
`type` is unused, can you rename it to `unused` or similar?

Reply:
I marked it as ``Py_UNUSED``